packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/edit/OvirtCredentialsEdit.tsx

```diff
@@ -165,6 +232,24 @@ export const OvirtCredentialsEdit: React.FC<EditComponentProps> = ({ secret, onC
           isDisabled={insecureSkipVerify}
         />
       </FormGroup>
+      {insecureSkipVerify ? (
```

Comment:
why do we need this check ?
do you want to hide the hint in some cases, what are this cases ? is there a reason to hide it in this cases ?

Reply:
Yes, just to hide this hint in case of ignoring the ca - since there is no need for a hint of how to set this ca in that case. Make the dialog less loaded with information.